nailgun/entities.py

```diff
@@ -6195,6 +6195,32 @@ def rh_cloud_inventory_sync(self, synchronous=True, timeout=None, **kwargs):
         response = client.post(self.path('rh_cloud/inventory_sync'), **kwargs)
         return _handle_response(response, self._server_config, synchronous, timeout)
 
+    def rh_cloud_fetch_last_report_log(self):
+        """Fetch latest report log.
+
+        :param kwargs: Arguments to pass to requests.
+        :returns: The server's response, with all JSON decoded.
+        :raises: ``requests.exceptions.HTTPError`` If the server responds with
+            an HTTP 4XX or 5XX message.
+        """
+        kwargs = {'headers': {"Accept": "application/json"}}  # shadow the passed-in kwargs
+        kwargs.update(self._server_config.get_client_kwargs())
+        url = f"{self._server_config.url}/foreman_inventory_upload/{self.id}/reports/last"
```

Comment:
```suggestion
        url = f'{self._server_config.url}/foreman_inventory_upload/{self.id}/reports/last'
```
For maintaining consistency of string in file